Cleaning-Spreadsheets.md

```diff
@@ -0,0 +1,37 @@
+# Cleaning Data with Spreadsheets
+
+## In the prep work for this class, the students learned:
+
+1. What Cleaning data is.
+1. What common techniques are used to clean data.
+1. How to apply filters, formatting, remove redundant data, and useful functions to clean data.
+1. The four types of dirty data
+
+## Announcements
+1. Always check with your program manager to see if there are any upcoming deadlines you should highlight!
+
+## Large Group Time (Instructor)
+
+### Topics That Require Careful Attention
+1. Q & A from the prep work and/or exercises.
+   1. Recommend walking through the exercises since the studio covers the same dataset.
+1. Review the data cleaning techniques and apply any additional strategies not used in the book.
+1. Focus on data cleaning concepts and best practices for spreadsheets.
+1. If you can think of a time when you were cleaning spreadsheets and applied any useful techniques not used in the book please share!
+
+## Studio (IA Notes)
+
+### Best Practices for ALL Studios
+1. Check in with EVERY student during EVERY studio.
+   1. This builds relationships and trust, and it helps you spot potential issues before they get too large.
+   1. Checking in does NOT mean a quick question like, "How are you doing? Need any help?" Instead, sit next to the student and ask specific questions about their code, their interpretation of the instructions, or their coding dreams.
+   1. Provide detailed feedback as often as possible. "Good job!" does not cut it. An example would be "Hey! I like your variable names! They are very descriptive!".
+1. Be prepared to clarify the studio instructions beyond just re-reading the words on the screen.
+1. Encourage students to work together and share ideas.
+1. Assist individuals as questions arise. Address frequent mistakes and/or questions to your whole group.
+1. Make a note of any issues that occur during the studio and provide that feedback to the instructor and LaunchCode team.
```

Comment:
I think we only really need this section on the first day of class wiki.

Reply:
Removed above section!